### PR TITLE
Cleanup ci

### PR DESCRIPTION
**Pull request by syyyr**

### CODE REVIEW

**Review by chinese-soup — COMMENTED** [review]
<where the ufck did this comment go so i can delete it llol??>

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -5,10 +5,6 @@ on:
     branches:
       - '**'
   create:
-    branches:
-      - 'master'
-    tags:
-      - '**'
```
Comment (chinese-soup) [inline .github/workflows/main.yml:11]: Why was this removed? I'd assume this is useful for Bizon to tag a release and then instantly use the artifacts from it for it to upload to TestFlight or something.

**Review by chinese-soup — COMMENTED** [review]
lgtm otherwise

File: .github/workflows/main.yml (hunk 1 above)
Comment (syyyr) [reply .github/workflows/main.yml:11]: As the [documentation](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushbranchestagsbranches-ignoretags-ignore) says (and *actionlint* agrees), the `tags` key can only be inside the `push` key. I didn't add it there, because I didn't want the CI to change, I just wanted to clean it up.

Comment (syyyr) [reply .github/workflows/main.yml:11]: Looking in the doc again, it seems that the `on.create` key is also invalid

**Review by MartinBriza — CHANGES_REQUESTED** [review]
This will break my flow of automatic releases when I push a new tag. Seems it should be possible to fix this by using the `on.push.tags` event.

File: .github/workflows/main.yml (hunk 1 above)
Comment (MartinBriza) [reply .github/workflows/main.yml:11]: This will change the CI though.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -82,12 +78,12 @@ jobs:
...
-        patch -p1 < $GITHUB_WORKSPACE/dist/ios/qt-ios.patch
+        pushd "$Qt6_DIR"
+        patch -p1 < "$GITHUB_WORKSPACE"/dist/ios/qt-ios.patch
         popd
-        echo -e "[Paths]\nPrefix=.." > $Qt6_DIR/bin/qt.conf
-        sed -i .old 's/clang_64/macos/' $Qt6_DIR/bin/target_qt.conf
-        sed -i .old 's/clang_64/macos/' $Qt6_DIR/bin/qmake
+        echo -e "[Paths]\nPrefix=.." > "$Qt6_DIR"/bin/qt.conf
+        sed -i .old 's/clang_64/macos/' "$Qt6_DIR"/bin/target_qt.conf
+        sed -i .old 's/clang_64/macos/' "$Qt6_DIR"bin/qmake
```
Comment (MartinBriza) [inline .github/workflows/main.yml:86]: TI KKS
